scripts/data_contamination/README.md

```diff
@@ -65,3 +65,21 @@ with beam.Pipeline() as pipeline:
 ## Notes
 
 The beam script does not support outputting contaminated ngrams yet.
+
+
+## Docker
+
+To create and run docker image:
+
+docker build  . -t contamination_script
+docker run -e INPUT_PATH=<input_path>... SCENARIO_DATA... OUTPUT_STATS... INPUT_FORMAT --rm -it  contamination_script:latest 
```

Comment:
It's a bit unclear what the args should be...I'd put the example here, and take the defaults out of the Dockerfile

Reply:
Sounds good, removed defaults above and added example here